data/yam/agama/auto/lib/security.libsonnet

```diff
@@ -0,0 +1,10 @@
+{
+  sslCertificates(fingerprint=''):: {
+    sslCertificates: [
+      {
+        fingerprint: fingerprint,
```

Comment:
we don't need to pass this parameter from outside, doesn't seem to change and if it does, we can update it in single place. We can fix it here.

Reply:
Changed to real code.